src/webparts/scw/components/SCW.tsx

```diff
@@ -516,7 +525,7 @@ export default class SCW extends React.Component<ISCWProps, ISCWState> {
                     <p>{strings.congratPara1}</p>
                     <p aria-live="polite">{strings.congratPara2}</p>
                     <button autoFocus onClick={() => this.ResetScreen()}>{strings.congratHome}</button>
```

Comment:
When referring to changing the home link, I believe they also wanted the Home button to also link back to home and not the first step of the SCW.

Also we should have the support link link directly to the support hub.